docs/source/howto/plugin_codes.rst

```diff
@@ -223,104 +250,205 @@ The Topics section on :ref:`defining processes <topics:processes:usage:defining>
 
 .. _how-to:plugin-codes:run:
 
-Running external codes
-======================
+Running your calculation
+========================
 
-To run an external code with AiiDA, you will need to use an appropriate :ref:`calculation plugin <topics:plugins>` that knows how to transform the input nodes into the input files that the code expects, copy everything in the code's machine, run the calculation and retrieve the results.
-You can check the `plugin registry <https://aiidateam.github.io/aiida-registry/>`_ to see if a plugin already exists for the code that you would like to run.
-If that is not the case, you can :ref:`develop your own <how-to:plugin-codes:interfacing>`.
-After you have installed the plugin, you can start running the code through AiiDA.
-To check which calculation plugins you have currently installed, run:
+With your ``calcjob.py`` and ``parser.py`` files at hand, you can launch your first calculation:
 
-.. code-block:: bash
 
-    $ verdi plugin list aiida.calculations
+ 1. If you haven't already done so, set up your localhost computer:
 
-As an example, we will show how to use the ``arithmetic.add`` plugin, which is a pre-installed plugin that uses the `bash shell <https://www.gnu.org/software/bash/>`_ to sum two integers.
-You can access it with the ``CalculationFactory``:
+    .. code-block:: console
 
-.. code-block:: python
+        $ verdi computer setup -L localhost -H localhost -T local -S direct -w `echo $PWD/work` -n
+        $ verdi computer configure local tutor --safe-interval 5 -n
```

Comment:
have you tested these commands? I don't think `tutor` exists. Think that should be `localhost`.
Shouldn't this also just refer to a section that explains how to setup a computer?

Reply:
That should have been `verdi computer configure local localhost ...` 
I agree that we can refer to the computer setup section here, though.
Will change this.

Reply:
```suggestion
        $ verdi computer configure local localhost --safe-interval 5 -n
```